apps/android-service-runtime/src-tauri/src/lib.rs

```diff
@@ -13,7 +14,11 @@ pub fn run() {
     log::set_max_level(LevelFilter::Warn);
 
     tauri::Builder::default()
-        .plugin(tauri_plugin_holochain_service::init())
+        .plugin(tauri_plugin_holochain_service::init(RuntimeNetworkConfigFfi {
+            bootstrap_url: "https://bootstrap-0.infra.holochain.org".to_string(),
+            signal_url: "wss://sbd.holo.host".to_string(),
+            ice_urls: vec!["stun:stun.l.google.com:19302".to_string()]
+        }))
```

Comment:
The runtime network config is now specified by the app using `tauri-plugin-holochain-service`, rather than being hard-coded.